migrations/2016_01_01_000000_add_voyager_user_fields.php

```diff
@@ -9,8 +9,8 @@ class AddVoyagerUserFields extends Migration
      */
     public function up()
     {
-        Schema::table('users', function ($table) {
-            if (!Schema::hasColumn('users', 'avatar')) {
+        Schema::table(get_prefixed_table('users'), function ($table) {
```

Comment:
We **_really_** shouldn't be changing old migrations... Works well for new users, but could very easily burn existing users trying to upgrade.

Reply:
I understand your concern but I don't think we would be able to add prefix support without changing old migrations.

If that can help whenever you are updating or rolling back `get_prefix_table` helper will output the same table name passed if voyager.database.table_prefix is not set (as it should for previous installations) or empty (default value for new installations).

So, unless I'm missing something, it shouldn't change anything for old users unless they willingly add a table_prefix.